packages/hint-manifest-icons/src/hint.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * @fileoverview Checks usage of Web App Manifest icons
+ */
+import { URL } from 'url';
+import * as getImageData from 'image-size';
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+// The list of types depends on the events you want to capture.
+import { IHint, NetworkData } from 'hint/dist/src/lib/types';
+import {
+    ManifestEvents,
+    ManifestParsed,
+    ManifestImageResource
+} from '@hint/parser-manifest';
+import { debug as d } from '@hint/utils';
+
+import meta from './meta';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class ManifestIconHint implements IHint {
+    public static readonly meta = meta;
+    public constructor(context: HintContext<ManifestEvents>) {
+        /**
+         * Try to see if the `icon` file actually
+         * exists and is accessible.
+         */
+        const iconExists = async (IconPath: string) => {
```

Comment:
```suggestion
        const iconExists = async (iconPath: string) => {
```

You'll have to rename it everywhere.

Reply:
Done.